src/Stratis.Bitcoin/P2P/PeerConnectorAddNode.cs

```diff
@@ -83,7 +83,7 @@ public override async Task OnConnectAsync()
                         return;
 
                     PeerAddress peerAddress = this.peerAddressManager.FindPeer(ipEndpoint);
-                    if (peerAddress != null && !this.IsPeerConnected(peerAddress.Endpoint))
```

Comment:
Why remove `IsPeerConnected`?

Reply:
This is now checked in the base class :)